mk/top_of_makefile.mk

```diff
@@ -39,14 +41,16 @@ endef
 $(error TARGET must be of the form \
         <arch>[<sub>]-<vendor>-<sys>-<abi>.$(NEWLINE)\
 \
-\       Exceptions: <abi> defaults to "macho" on Mac OS X.\
-\
+        Exceptions: <abi> defaults to "macho" on Mac OS X \
+        and to "androideabi" on Android. $(NEWLINE)\
```

Comment:
Again, `<vendor>` defaults to `unknown` on Android, and the ABI is explicitly given.